Middleware/SVGElements/Lightning.cs

```diff
@@ -0,0 +1,93 @@
+using System;
+
+// TODO: Move this too
+
+// This is PRNG based on the system clock. Good enough for random lightning strikes, but if you need
+// better randomness you should use another class.
+public class RandomGenerator
+{
+    private static readonly Random _random = new Random();
```

Comment:
Use Random.Shared for a thread safe random object.. Don't need to keep an instance around.

https://learn.microsoft.com/en-us/dotnet/api/system.random.shared?view=net-8.0